src/components/Listing/index.js

```diff
@@ -0,0 +1,16 @@
+import React from "react";
```

Comment:
Does this page represent one single listing or a list of all listings (paginated possibly)?